ontologies/Journal_with_items.md

```diff
@@ -0,0 +1,35 @@
+
+#JSON examples journals
+
+Comment line 13-22: 
+a) are we happy dealing with all these different sorts of title as subsets of alternativeTitlt?
+b) if so, keyTitle and uniformTitle need to be added to the ontology and described
+
+Comment line 43: "designation" needs to be added the ontology
+It corresponds in part to the start date and end date properties that are already there, covering dates of publication and sequential designation, but has other things mixed in like volumes
+
+Comment line 44 onwards: ISSN is not in the ontology yet
+It occupies the same field as ISBN and the distinguishing feature is apparently the format: ISSN has four digits, a slash then another four digits.
```

Comment:
ISSN/ISBN luckily do not occupy the same field. 020 is for ISBN. 022 is for ISSN.  http://www.loc.gov/marc/bibliographic/bd01x09x.html